application/src/main/java/org/togetherjava/tjbot/commands/moderation/temp/RevocableModerationAction.java

```diff
@@ -11,23 +12,6 @@
  * {@link TemporaryModerationRoutine} to identify and revoke such actions.
  */
 interface RevocableModerationAction {
-    /**
-     * Classification of a revocation failure.
-     */
-    @SuppressWarnings("PublicInnerClass")
-    enum FailureIdentification {
-        /**
-         * Acknowledges that the failure is known and has been handled. Hence, further error
-         * handling should not be continued.
-         */
-        KNOWN,
-        /**
-         * The failure is unknown and has not been handled. Hence, further error handling should be
-         * continued.
-         */
-        UNKNOWN
-    }
```

Comment:
Why did you remove this? Please undo this. Now you reverted to "boolean" and the code is now worse, you landed in the so called "boolean-trap". Returning an enum is by far more readable to an user than a boolean. Please use the enum instead again, it was put there on purpose to avoid the boolean-trap.

Reply:
undid it.